daffodil-cli/src/main/scala/org/apache/daffodil/cli/InfosetTypes.scala

```diff
@@ -71,15 +71,15 @@ import org.xml.sax.helpers.DefaultHandler
 object InfosetType extends Enumeration {
   type Type = Value
 
-  val EXI = Value("exi")
```

Comment:
The `SpecifyReturnType` fix seems to add a lot of extra verbosity. I assume this is just a style change and not something deprecated by scala 2.13 or 3.x? There may be some places where having an explicit return type might be useful, but I'm not sure we should require it everywhere if future scala versions don't require it.

Thoughts?

Reply:
Looking at the old PR, I assumed return types were required in 2.13. I'll do more research and see

https://github.com/apache/daffodil/pull/246

Reply:
So it looks like it's recommended but not strictly enforced from https://docs.scala-lang.org/scala3/guides/migration/incompatibility-table.html#type-inference

`It is always good practice to write the result types of all public values and methods explicitly. It prevents the public API of your library from changing with the Scala version, because of different inferred types. This can be done prior to the Scala 3 migration by using the [ExplicitResultTypes](https://scalacenter.github.io/scalafix/docs/rules/ExplicitResultTypes.html) rule in Scalafix.`

Reply:
According to the Scala 3 book: https://docs.scala-lang.org/scala3/book/methods-most.html

It says 

> Declaring the method return type is optional

and

> It is recommended to annotate publicly visible methods with their return type. Declaring the return type can make it easier to understand it when you look at it months or years later, or when you look at another person’s code.

So I don't think it's required, but is just good practice. But we have so many public members that I think it would make things too verbose. We may should be better about marking things as private, but that's a whole other effort, and it's probably not worth it.